src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/trax/TemplatesImpl.java

```diff
@@ -438,6 +438,7 @@ public Set<ModuleReference> findAll() {
      * Defines the translet class and auxiliary classes.
      * Returns a reference to the Class object that defines the main class
      */
+    @SuppressWarnings("removal")
     private void defineTransletClasses()
         throws TransformerConfigurationException {
```

Comment:
In passing, the PermissionCollection used to construct the PD here is no longer needed. If this is cleaned up then the SW wouldn't be needed.